src/view/UserphraseView.cpp

```diff
@@ -75,6 +79,12 @@ void UserphraseView::addPhrase(int result)
     }
 
     emit model()->add(phrase, bopomofo);
+    
+    if (dialogType_ == DIALOG_MODIFY) {
+	if (model()->sourceModel()->getAddResult() <= 0) {
```

Comment:
What do you mean "consistent coding style"?
Is there any thing can be referenced?